doc/release/platforms/cray.rst

```diff
@@ -17,19 +17,36 @@ Getting Started with Chapel on Cray X-Series Systems
 
 Chapel is available as a module for Cray X-series systems.  When it is
 installed on your system, you do not need to build Chapel from the
-source release (though you can); instead, simply load the module
-using::
+source release (though you can). To use Chapel with the default settings and
+confirm it is correctly installed, do the following:
+
+1) Load the Chapel module::
 
      module load chapel
 
-Then proceed directly to `Using Chapel on a Cray System`_ below.  If instead
-you wish to build Chapel from source, continue on to `Building Chapel for a
-Cray System from Source`_ just below.
+
+2) Compile an example program using::
+
+     chpl -o hello6-taskpar-dist $CHPL_HOME/examples/hello6-taskpar-dist.chpl
+
+
+3) Execute the resulting executable (on four locales)::
+
+     ./hello6-taskpar-dist -nl 4
+
+
+This is all that is necessary for most Chapel installations on Cray X-Series
+systems.  If the installation setup by your system administratordeviates from
+the default settings, or you are interested in additional configuration
```

Comment:
Maybe "other configuration settings" rather than "additional configuration options".  It's more that you're setting some of them differently, than that there are more of them.